torch/testing/_internal/common_methods_invocations.py

```diff
@@ -280,6 +311,60 @@ def __init__(self,
         #   outside a function's domain.
         self._domain_eps = 1e-5
 
+    def sample_sparse_inputs(self, device, dtype, requires_grad=False, **kwargs):
```

Comment:
I think we can reuse the same pattern that sample_inputs() uses for the UnaryUfuncs? See comment above.